packages/frontend/src/components/screens/MainScreen/MainScreen.tsx

```diff
@@ -402,3 +436,33 @@ function ChatNavButtons({ chat }: { chat: T.FullChat }) {
     </>
   )
 }
+
+function AppIcons({
+  accountId,
+  apps,
+}: {
+  accountId: number | undefined
+  apps: T.Message[]
+}) {
+  if (!accountId || !apps || apps.length === 0) {
+    return null
+  }
+  return (
+    <div className={styles.webxdcIcons}>
+      {apps.map(app => (
+        <img
+          key={app.id}
+          className={styles.webxdcIcon}
+          src={runtime.getWebxdcIconURL(accountId, app.id)}
+          alt=''
```

Comment:
for accessibility we should also put the app name here?
```suggestion
          alt={app.webxdcInfo?.name}
```